app/src/main/java/com/dsu/poc/ui/login/model/LoginModel.kt

```diff
@@ -1,4 +1,20 @@
 package com.dsu.poc.project.ui.login.model
 
-class LoginModel {
+import androidx.databinding.BaseObservable
+import androidx.databinding.Bindable
+import com.dsu.poc.BR
+
+class LoginModel : BaseObservable() {
```

Comment:
why is this extending BaseObservable?

Reply:
All model classes should be a data class with constructor parameters. Please recheck this. You can learn about data classes here: https://medium.com/mindorks/data-classes-in-kotlin-369e95c2f0ea